cmd/collectors/zapi/plugins/quota/quota.go

```diff
@@ -222,3 +234,92 @@ func (my *Quota) Run(data *matrix.Matrix) ([]*matrix.Matrix, error) {
 	}
 	return output, nil
 }
+
+func (my *Quota) handle7Mode(request *node.Node, data *matrix.Matrix) ([]*matrix.Matrix, error) {
```

Comment:
most of the code looks similiar to 7 mode. If it's just idifferent attribute , can this be handled in same method which will avoid code duplication?

Reply:
Yes it is. Done.